test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -922,8 +922,9 @@ public static TheoryData InvalidTagHelperAttributeDescriptorData
                         new[] {
                             string.Format(
                                 errorFormat,
+                                typeof(InvalidBoundAttribute).FullName,
                                 nameof(InvalidBoundAttribute.DataSomething),
-                                typeof(InvalidBoundAttribute).FullName)
```

Comment:
Test fix reflects product code fix: Message previously included 'Property.Namespace.Contains.Class`.  Full type name and property name now included in correct order: `'Namespace.Contains.Class.Property`.